app/controllers/application_controller.rb

```diff
@@ -1,3 +1,13 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery
+
+  def require_user
+  	redirect_to sign_in_path unless current_user
```

Comment:
Make sure your editor is setup to use 2 spaces instead of tab characters. This does not appear to be the case right now.